products/BellHybrid/apps/application-bell-bedtime/ApplicationBellBedtime.cpp

```diff
@@ -1,21 +1,23 @@
-// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include "ApplicationBellBedtime.hpp"
 #include "presenter/BellBedtimeWindowPresenter.hpp"
 #include "windows/BellBedtimeWindow.hpp"
-#include "common/models/BedtimeModel.hpp"
+#include <common/models/BedtimeModel.hpp>
 #include <common/windows/BellFinishedWindow.hpp>
+#include <common/SoundsRepository.hpp>
 #include "models/BedtimeListItemProvider.hpp"
```

Comment:
You can move `models/BedtimeListItemProvider.hpp` before `<common/....>` includes because it is a part of the bedtime application.

Reply:
Done :wink: